### PR TITLE
BuntDB support resumed and fixed typos

### PR DESCRIPTION
**Pull request by gandaldf**
About #116 issue

### CODE REVIEW

**Comment by gandaldf** [comment]
Hi Alex! I've fixed the buntdbstore test and thanks to your answer (#117) I've implemented the All() method in some of the stores, I hope you will find it useful!

**Comment by alexedwards** [comment]
This is fantastic, thank you so much! :smiley: 

It all looks good to me, so I'll merge it now.

File: buntdbstore/buntdbstore.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
+package buntdbstore
```
Comment (Adamthefirstmantogetlaid) [inline buntdbstore/buntdbstore.go:1]: What's up

**Review by Adamthefirstmantogetlaid — COMMENTED** [review]
Thefirstadamtogetlaid

**Pull request merged**